README.md

```diff
@@ -1,6 +1,17 @@
 # MII feasibility Structured Query
 
-Includes 
+## Includes 
 * an example of a structured query (as JSON file)
 * a JSON schema describing a structured query
 * [documenation as .md-file](documentation/2021_10_18StructeredQueryV2Documentation.md) 
+* a python script that generates structured queries that fit to custom testdata
+
+## Usage of Structured Query Generator
+* the testdata should be placed in a json file in the form of an array of FHIR resources like [here](test/resources/testdata.json) 
+* the default testdata file is `test/resources/testdata.json` but you can specifiy a custom filename with the -i option
+* the resulting structured queries will be saved to `sq-generator/generated-sqs/` by default, but you can specify it with the -o option
```

Comment:
As default, I would only use the current working directory, not something specific.